demo.Dockerfile

```diff
@@ -1,7 +1,15 @@
+FROM jaegertracing/all-in-one:latest AS jaeger-builder
```

Comment:
I don't think we need this line here, the binary should already be available in the image `quay.io/konveyor/analyzer-lsp`